Automattic/configure
# Add Ruby Gem

Adds a Ruby Gem to the project that can be used elsewhere. It uses FFI to interface with the library target.

**To Test:**
1. Check out the WordPress iOS project
2. Update the `Gemfile` to add `gem 'libconfigure', '~> 0.3.4'`
3. Run `bundle install`
4. Edit the `.configure` file's `project_name` field to `wordpress-ios` and save the file
5. Ensure your local mobile secrets are up-to-date
6. Run `bundle exec configure_update` and follow the prompts to get the latest secrets
7. Delete the files that don't end in `.enc` in `.configure-files`, and run `bundle exec configure_apply` – ensure the files are re-created.
